### ISSUE TITLE
FutureWarning: expected range[0,1] tensor in SVD pipeline

### ISSUE THREAD

**Issue by shliu0** [huggingface/diffusers#6574]
### Describe the bug

Why set this warning in [diffusers/image_processor.py:339:](https://github.com/huggingface/diffusers/blob/76c645d3a641c879384afcb43496f0b7db8cc5cb/src/diffusers/image_processor.py#L339C1-L343C14) 
`FutureWarning: Passing `image` as torch tensor with value range in [-1,1] is deprecated. The expected value range for image tensor is [0,1] when passing as pytorch tensor or numpy Array. `
Is there an error in svd pipeline, if I pass image as torch.FloatTensor to pipeline, it will use its [CLIP preprocess](https://github.com/huggingface/diffusers/blob/5bacc2f5af7904749827fac15df05df0cb782fe4/src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py#L424C1-L424C1) and [VAE preprocess](https://github.com/huggingface/diffusers/blob/5bacc2f5af7904749827fac15df05df0cb782fe4/src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py#L432C3-L432C4). While [CLIP preprocess](https://github.com/huggingface/diffusers/blob/5bacc2f5af7904749827fac15df05df0cb782fe4/src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py#L118C1-L136C27) require the input tensor to be CLIP normalized image with size(224,224), 
 [VAE preprocess](https://github.com/huggingface/diffusers/blob/76c645d3a641c879384afcb43496f0b7db8cc5cb/src/diffusers/image_processor.py#L336C1-L347C42) require the input tensor to be [0,1] normalized image with size(576, 1024), aren't these two requirements contradictory?

### Reproduction

```
import torch
from diffusers import StableVideoDiffusionPipeline
from diffusers.pipelines.stable_video_diffusion.pipeline_stable_video_diffusion import _resize_with_antialiasing
from diffusers.utils import load_image, export_to_video

pipe = StableVideoDiffusionPipeline.from_pretrained(
    "stabilityai/stable-video-diffusion-img2vid-xt", torch_dtype=torch.float16, variant="fp16"
).to("cuda:0")

# Load the conditioning image
image = load_image("https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/svd/rocket.png")
image = image.resize((1024, 576))
image = pipe.image_processor.pil_to_numpy(image)  # to np
image = pipe.image_processor.numpy_to_pt(image)  # to pt
image = image * 2.0 - 1.0  # normalize to [-1, 1]
image = _resize_with_antialiasing(image, (224, 224))
image = (image + 1.0) / 2.0 # [-1, 1] -> [0, 1]

# image = pipe.feature_extractor(
#                 images=image,
#                 do_normalize=True,
#                 do_center_crop=False,
#                 do_resize=False,
#                 do_rescale=False,
#                 return_tensors="pt",
#             ).pixel_values

generator = torch.manual_seed(42)
frames = pipe(image, decode_chunk_size=8, generator=generator).frames[0]
export_to_video(frames, "generated.mp4", fps=7)
```

### System Info

- `diffusers` version: 0.24.0
- Platform: Linux-3.10.0-1160.el7.x86_64-x86_64-with-glibc2.17
- Python version: 3.10.13
- PyTorch version (GPU?): 2.0.1+cu118 (True)
- Huggingface_hub version: 0.19.4
- Transformers version: 4.30.0
- Accelerate version: 0.21.0
- xFormers version: 0.0.22
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No

**Comment by patrickvonplaten** [comment]
Hey @shliu0,

Could you please make sure to post a fully reproducible code snippet?

**Comment by hi-sushanta** [comment]
Could you please provide your full code? I think your image already normalizes because it goes down the first if statement that provides a warning message and does not normalize the image.

**Comment by shliu0** [comment]
Thanks for your attention! I have update a toy example. 
About my first question, I understand that the warning is raised because the input image is already normalized, but what I don't understand is why this warning is set, and why the new version of the diffusers code does not support normalized images as input.
My key point is my 2nd question, `Is there an error in svd pipeline, if I pass image as torch.FloatTensor to pipeline, it will use its [CLIP preprocess](https://github.com/huggingface/diffusers/blob/5bacc2f5af7904749827fac15df05df0cb782fe4/src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py#L424C1-L424C1) and [VAE preprocess](https://github.com/huggingface/diffusers/blob/5bacc2f5af7904749827fac15df05df0cb782fe4/src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py#L432C3-L432C4). While [CLIP preprocess](https://github.com/huggingface/diffusers/blob/5bacc2f5af7904749827fac15df05df0cb782fe4/src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py#L118C1-L136C27) require the input tensor to be CLIP normalized image with size(224,224),
[VAE preprocess](https://github.com/huggingface/diffusers/blob/76c645d3a641c879384afcb43496f0b7db8cc5cb/src/diffusers/image_processor.py#L336C1-L347C42) require the input tensor to be [0,1] normalized image with size(576, 1024), aren't these two requirements contradictory?`
According to [this](https://github.com/huggingface/diffusers/blob/5bacc2f5af7904749827fac15df05df0cb782fe4/src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py#L128C1-L136C27), if I want to pass _images_ as **torch.Tensor** to pipe, the blowing code(CLIP normalize) is necessary, right? However, this might cause color distortion in generated video.
`# image = pipe.feature_extractor(
#                 images=image,
#                 do_normalize=True,
#                 do_center_crop=False,
#                 do_resize=False,
#                 do_rescale=False,
#                 return_tensors="pt",
#             ).pixel_values`
Below is my results of comment & uncomment the code above.

https://github.com/huggingface/diffusers/assets/126319446/4071fd63-2d54-4b76-a3fd-1fe593e96436


https://github.com/huggingface/diffusers/assets/126319446/7313efa6-afa7-416d-a84d-fd9b599eb689

**Comment by yiyixuxu** [comment]
hey @shliu0 
thanks for reporting this

I think it is indeed a bug and we are fixing it here https://github.com/huggingface/diffusers/pull/6999

note that you don't need to preprocess the image tensor with CLIPImageProcessor before passing it to the pipelines

**Comment by shliu0** [comment]
> hey @shliu0 thanks for reporting this
> 
> I think it is indeed a bug and we are fixing it here #6999
> 
> note that you don't need to preprocess the image tensor with CLIPImageProcessor before passing it to the pipelines

Thanks! If there is also a need to move [these code](https://github.com/huggingface/diffusers/blob/5bacc2f5af7904749827fac15df05df0cb782fe4/src/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py#L124C1-L126C40) outside the `if not isinstance(image, torch.Tensor):` branch, cause CLIP image_encoder is trained on [(224,224) crop](https://huggingface.co/lambdalabs/sd-image-variations-diffusers/blob/42bc0ee1726b141d49f519a6ea02ccfbf073db2e/feature_extractor/preprocessor_config.json#L2)?
